client/src/api/profiles.js

```diff
@@ -1,3 +1,11 @@
-export const getProfiles = () => {
-  return fetch("/api/profiles").then(res => res.json());
+export const getProfiles = profile => {
+  const sendProfile = {
```

Comment:
It looks like this variable isn't being used. Should it be used in the `fetch` call below?